test/parallel/test-tls-cipher-list.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+var common = require('../common');
+
+if (!common.hasCrypto) {
+  console.log('1..0 # Skipped: missing crypto');
+  return;
+}
+
+var assert = require('assert');
+var spawn = require('child_process').spawn;
+const default_core_list = require('constants').defaultCoreCipherList;
+
+function doCheck(arg, env, check) {
+  var out = '';
+  var arg = arg.concat([
+    '-pe',
+    'require("constants").defaultCipherList'
+  ]);
+  spawn(process.execPath, arg, env).
+     on('error', assert.fail).
```

Comment:
Probably you can take care of this while landing, indentation of on is off by one space.